code/game/turfs/simulated/wall_icon.dm

```diff
@@ -94,30 +94,23 @@
 		img.alpha = (i * alpha_inc) - 1
 		damage_overlays[i] = img
 
-
 /turf/simulated/wall/proc/update_connections(propagate = 0)
 	if(!istype(material))
 		return
 	var/list/dirs = list()
-<<<<<<< HEAD
-	var/inrange = orange(src, 1)

```

Comment:
I think keeping inrange at least would be better here, to not call orange multiple times down the line.